### ISSUE TITLE
<svelte:options/> unknown node type: Options

### ISSUE THREAD

**Issue by frederikhors** [UnwrittenFun/svelte-vscode#59]
Using

```html
<svelte:options immutable/>
```

the formatting gives error in VSCode's output console:

```
[Error - 17:10:11] Request textDocument/formatting failed.
  Message: Request textDocument/formatting failed with message: unknown node type: Options
  Code: -32603 
```

**Comment by frederikhors** [comment]
Sorry, I saw this just now: https://github.com/UnwrittenFun/prettier-plugin-svelte/pull/42 but it seems still present.

Why?

**Comment by jamesbirtles** [comment]
should be fixed in v0.9.2
